src/Pester.Types.ps1

```diff
@@ -1,6 +1,20 @@
 if ($PSVersionTable.PSVersion.Major -ge 6) {
-    & $SafeCommands['Add-Type'] -Path "$PSScriptRoot/bin/netstandard2.0/Pester.dll"
+    $path = "$PSScriptRoot/bin/netstandard2.0/Pester.dll"
+    if ((Get-Variable -Name "PESTER_BUILD" -ValueOnly -ErrorAction Ignore)) {
+        $path = "$PSScriptRoot/../bin/bin/netstandard2.0/Pester.dll"
+    }
+    else {
+        $path = "$PSScriptRoot/../bin/bin/netstandard2.0/Pester.dll"
+    } # endif
+    & $SafeCommands['Add-Type'] -Path $path
 }
 else {
-    & $SafeCommands['Add-Type'] -Path "$PSScriptRoot/bin/net452/Pester.dll"
+    $path = "$PSScriptRoot/bin/net452/Pester.dll"
+    if ((Get-Variable -Name "PESTER_BUILD" -ValueOnly -ErrorAction Ignore)) {
+        $path = "$PSScriptRoot/../bin/bin/net452/Pester.dll"
+    }
+    else {
+        $path = "$PSScriptRoot/../bin/bin/net452/Pester.dll"
+    } # endif
+    & $SafeCommands['Add-Type'] -Path $path
```

Comment:
quick follow-up: what is the idea with the inner else-blocks (L6-L8 and L16-18)?